### PR TITLE
Change path divide to be along y-axis

### PR DESCRIPTION
**Pull request by inchkev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,13 +26,9 @@ __Note: This project supports Python version 3.x.x.__
 
 To start the application from the project root, run:
 ```bash
-python copis/client.py
-```
-Or, using the start script in that folder, run:
-```bash
 python copisclient.py
 ```
-*Note: These 2 commands can be run from any folder in the project directory tree.*
```
Comment (djihbril) [inline README.md:35]: just curious why  you removed this line?

Comment (inchkev) [reply README.md:35]: Oh, I deleted it because there's just one command now - but I can put in a note saying you can run copisclient from anywhere

Comment (djihbril) [reply README.md:35]: actually leave the readme alone. on my branch, I have this:

### Usage

To start the application from the project root, run:
```bash
python copisclient.py
```
Or, as a module:
```bash
python -m copisclient
```
*Note: These 2 commands can be run from any folder in the project directory tree.*

Comment (inchkev) [reply README.md:35]: sure - i'll revert my changes

Comment (inchkev) [reply README.md]: done!

**Review by djihbril — APPROVED** [review]

**Pull request merged**